mozilla/fx-private-relay
# MPP-2389: Fix mobile layout issue for phones onboarding page on Firefox for Android/iOS

## Summary

- Adjust homepage toggle menu for iOS display issue
- Fix screen width issue on Firefox for iOS/Android

This PR fixes [MPP-2389](https://mozilla-hub.atlassian.net/browse/MPP-2389)

How to test:

- Use Firefox for Android or Firefox for iOS
- Use account that has phones flag enabled, but has not yet registered a phone/started onboarding flow
- Go to /phone/ 
- **Expected:** The page content should be full-width, and not push past the viewport. 

### Checklist: 

### Screenshot

![image](https://user-images.githubusercontent.com/2692333/193977915-c0d9e76a-f371-473c-a168-312d3022e404.png)
